systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -271,102 +274,227 @@ void testCustomAndUpdatedValues() {
+        checkContainerConfiguration(zookeeperStatefulSetName(CLUSTER_NAME), "zookeeper", "ZOOKEEPER_CONFIGURATION",
+                "autopurge.purgeInterval=1\ntickTime=2100\ninitLimit=6\nsyncLimit=3\n");
+        checkReadinesLivenessProbe(zookeeperStatefulSetName(CLUSTER_NAME), "tls-sidecar", updatedInitialDelaySeconds, updatedTimeoutSeconds);
+
+        LOGGER.info("Getting entity operator to check configuration of TO and UO");
+        checkReadinesLivenessProbe(entityOperatorDeploymentName(CLUSTER_NAME), "topic-operator", updatedInitialDelaySeconds, updatedTimeoutSeconds);
+        checkReadinesLivenessProbe(entityOperatorDeploymentName(CLUSTER_NAME), "user-operator", updatedInitialDelaySeconds, updatedTimeoutSeconds);
+        checkReadinesLivenessProbe(entityOperatorDeploymentName(CLUSTER_NAME), "tls-sidecar", updatedInitialDelaySeconds, updatedTimeoutSeconds);
+    }
+
+    /**
+     * Verifies readinessProbe and livenessProbe properties in expected container
+     * @param podName Name of pod where container is located
+     * @param containerName The container where verifying is expected
+     * @param initialDelaySeconds expected value for property initialDelaySeconds
+     * @param timeoutSeconds expected value for property timeoutSeconds
+     */
+    void checkReadinesLivenessProbe(String podName, String containerName, int initialDelaySeconds, int timeoutSeconds) {
+        LOGGER.info("Getting pods by name {}", podName);
+        List<Pod> pods = kubeClient().listPods()
+                .stream().filter(p -> p.getMetadata().getName().startsWith(podName))
+                .collect(Collectors.toList());
+
+        if (pods.size() != 0) {
+            LOGGER.info("Testing configuration for container {}", containerName);
+            pods.forEach(pod -> {
+                pod.getSpec().getContainers().stream().filter(c -> c.getName().equals(containerName))
+                    .forEach(container -> {
+                        assertEquals(initialDelaySeconds, container.getLivenessProbe().getInitialDelaySeconds());
+                        assertEquals(initialDelaySeconds, container.getReadinessProbe().getInitialDelaySeconds());
+                        assertEquals(timeoutSeconds, container.getLivenessProbe().getTimeoutSeconds());
+                        assertEquals(timeoutSeconds, container.getReadinessProbe().getTimeoutSeconds());
+                    });
+            });
+        } else {
+            fail("Pod with name" + podName + " not found");
+        }
+    }
+
+    /**
+     * Verifies container configuration by environment key
+     * @param podName Name of pod where container is located
+     * @param containerName The container where verifying is expected
+     * @param configKey Expected configuration key
+     * @param config Expected configuration
+     */
+    void checkContainerConfiguration(String podName, String containerName, String configKey, String config) {
```

Comment:
Same comment about naming of `podName` parameter.

Also since you have the same logic for getting the `List<Pod> pods = ...` you should factor out a common method.